server/src/models/teamDistribution.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  Index,
+  ManyToOne,
+  OneToMany,
+  ManyToMany,
+} from 'typeorm';
+import { Course } from './course';
+import { CourseTask } from './courseTask';
+import { Student } from './student';
+import { Team } from './team';
+
+@Entity()
+export class TeamDistribution {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @CreateDateColumn()
+  createdDate: Date;
+
+  @UpdateDateColumn()
+  updatedDate: Date;
+
+  @ManyToOne(_ => Course, { nullable: true })
+  course: Course;
+
+  @Column({ nullable: true })
+  @Index()
+  courseId: number;
+
+  @Column({ type: 'timestamptz', nullable: true })
+  distributionStartDate: null | Date;
```

Comment:
`distribution` looks redundant as it's already in the name

Reply:
fixed